utilities/merge_operators/string_append/stringappend_test.cc

```diff
@@ -259,7 +271,7 @@ TEST_P(StringAppendOperatorTest, SimpleTest) {
 }
 
 TEST_P(StringAppendOperatorTest, SimpleDelimiterTest) {
-  auto db = OpenDb('|');
```

Comment:
For the test, can you keep all the original tests and add the new test that uses the string? The old interface is still in use and we do not want to remove the tests until the interface is deprecated. Also, keep the std::shared_ptr<DB> OpenTtlDb(char delim_char).

Reply:
But the old interface is still used if the specified string is of length 1, addressing the problem @mrambacher pointed at. 

It's a bit tricky to write completely independent tests using the new interface there since they are all parameterized and use two different database opening routines depending on the parameter in their `SetUp`. I believe it will complicate the setup step too much.